Dockerfile

```diff
@@ -28,11 +28,12 @@ RUN sudo apt-get update && \
     sudo rm -rf /var/lib/apt/lists/* && \
     sudo add-apt-repository ppa:ethereum/ethereum && \
     sudo apt-get update && \
-    sudo apt-get install solc -y
+    sudo apt-get install solc -y && \
```

Comment:
For this part, I checked my own help doc for installation about  new server:
sudo apt install build-essential
    - sudo apt-get install libssl-dev
    - sudo apt-get install cmake
    - sudo apt-get install pkg-config
    - sudo apt install apache2-utils
    - sudo apt install clang 
    
    so this is for prover node only now so for the solc, maybe no need.
    
    I am not sure whether other lib need or not, but if it pass the test I guess it is oko